### PR TITLE
CY-2150 Don't update execution token

### PR DESCRIPTION
**Pull request by tehasdf**
It's not stored, so no need to update it

### CODE REVIEW

File: rest-service/manager_rest/resource_manager.py [hunk 1]
```diff
@@ -550,12 +550,6 @@ def _reset_operations(self, execution, execution_token, from_states=None):
                     continue
 
                 update_operation = False
```
Comment (tyacbovi) [inline rest-service/manager_rest/resource_manager.py]: We can change it back to how this code was before the last commit here (update_operation is always false now)

File: tests/integration_tests/tests/agentless_tests/test_resume.py [hunk 2]
```diff
@@ -335,6 +335,7 @@ def test_force_resume_cancelled_nonresumable(self):
         self.assertEqual(execution.status, 'cancelled')
 
         self._unlock_operation('interface1.op_nonresumable')
+        time.sleep(5)
```
Comment (tyacbovi) [inline tests/integration_tests/tests/agentless_tests/test_resume.py]: well it's just for we will not see the unauthorized error in the log?

File: rest-service/manager_rest/resource_manager.py (hunk 1 above)
Comment (inbala) [inline rest-service/manager_rest/resource_manager.py]: `update_operation` can be removed and the `self.sm.update` can be back under the `if operation.state in from_states`

File: rest-service/manager_rest/resource_manager.py [hunk 3]
```diff
@@ -550,12 +550,6 @@ def _reset_operations(self, execution, execution_token, from_states=None):
...
-                kwargs = operation.parameters.get('task_kwargs', {}) \
```
Comment (inbala) [inline rest-service/manager_rest/resource_manager.py]: Also, `update_operation` can be removed and the `self.sm.update` can be back under the `if operation.state in from_states` like it was before my change.

File: tests/integration_tests/tests/agentless_tests/test_resume.py (hunk 2 above)
Comment (inbala) [inline tests/integration_tests/tests/agentless_tests/test_resume.py]: I think it happens in more tests :/

File: rest-service/manager_rest/resource_manager.py (hunk 1 above)
Comment (tehasdf) [reply rest-service/manager_rest/resource_manager.py]: it's not always false (also set by the other thin) but you're right, changed :P

File: tests/integration_tests/tests/agentless_tests/test_resume.py (hunk 2 above)
Comment (tehasdf) [reply tests/integration_tests/tests/agentless_tests/test_resume.py]: pretty much

we note that also one test already had a sleep after unlock_operation, and tbh every one should do that, so unlock_operation itself should do the sleep

Comment (tehasdf) [reply tests/integration_tests/tests/agentless_tests/test_resume.py]: yep, every test that uses _unlock_operation, so i put the sleep there

(one test already had it)

**Pull request merged**